RELEASE_NOTES.md

```diff
@@ -0,0 +1,5 @@
+#### 1.0.1 May 27 2016 ####
+Removed non-generic IPublisher and ISubscriber interfaces.
```

Comment:
Hi Victor, it was my mistake in not publishing the initial release as `beta`. For now there are no public libraries referencing `Reactive.Streams`. We just merged akka-streams into the main branch, but it won't be published/released for a few weeks.